bucket/cv4pve-pepper.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.5.0",
+    "description": "Launching SPICE remote-viewer to access VM running on Proxmox VE.",
+    "homepage": "https://github.com/Corsinvest/cv4pve-pepper",
+    "license": "GPL-3.0-only",
+    "depends": "extras/virt-viewer",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Corsinvest/cv4pve-pepper/releases/download/v1.5.0/cv4pve-pepper.exe-win-x64.zip",
+            "hash": "9922f4a698457c024905c17506ec69d7b6adabdc035206c331c6b0c2583133ab"
+        },
+        "32bit": {
+            "url": "https://github.com/Corsinvest/cv4pve-pepper/releases/download/v1.5.0/cv4pve-pepper.exe-win-x86.zip",
+            "hash": "8c721843c5acd51629c872a710f7928fb4d1438313d19681333f5cecd5887c30"
+        }
+    },
+    "bin": "cv4pve-pepper.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Corsinvest/cv4pve-pepper/releases/download/v$version/cv4pve-pepper.exe-win-x64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/Corsinvest/cv4pve-pepper/releases/download/v$version/cv4pve-pepper.exe-win-x86.zip"
+            }
+        }
+    }
+}
```

Comment:
Do we not need to persist anything?

Reply:
It has no data or config that need to persist.